src/commands/lm/setup.js

```diff
@@ -0,0 +1,115 @@
+const { flags: flagsLib } = require('@oclif/command')
+const execa = require('execa')
+const Listr = require('listr')
+
+const Command = require('../../utils/command')
+const { installPlatform } = require('../../utils/lm/install')
+const { checkHelperVersion } = require('../../utils/lm/requirements')
+const { printBanner } = require('../../utils/lm/ui')
+
+const installHelperIfMissing = async function (force) {
+  let installHelper = false
+  try {
+    const version = await checkHelperVersion()
+    if (!version) {
+      installHelper = true
+    }
+  } catch (error) {
+    installHelper = true
+  }
+
+  if (installHelper) {
+    return installPlatform(force)
+  }
+
+  return false
+}
+
+const provisionService = async function (siteId, api) {
+  const addonName = 'large-media'
+
+  if (!siteId) {
+    return Promise.reject(new Error('No site id found, please run inside a site folder or `netlify link`'))
+  }
+  try {
+    await api.createServiceInstance({
+      siteId,
+      addon: addonName,
+      body: {},
+    })
+  } catch (error) {
+    // error is JSONHTTPError
+    return Promise.reject(new Error(error.json.error))
+  }
+}
+
+const configureLFSURL = async function (siteId, api) {
+  const siteInfo = await api.getSite({ siteId })
+  const url = `https://${siteInfo.id_domain}/.netlify/large-media`
+
+  return execa('git', ['config', '-f', '.lfsconfig', 'lfs.url', url])
+}
+
+class LmSetupCommand extends Command {
+  async run() {
+    await this.authenticate()
+
+    const { flags } = this.parse(LmSetupCommand)
+    const { api, site } = this.netlify
+
+    await this.config.runHook('analytics', {
+      eventName: 'command',
+      payload: {
+        command: 'lm:setup',
+      },
+    })
+
+    let helperInstalled = false
+    if (!flags['skip-install']) {
+      try {
+        helperInstalled = await installHelperIfMissing(flags['force-install'])
+      } catch (error_) {
+        this.error(error_.message)
```

Comment:
```suggestion
        this.error(error_.message)
```

To keep the stack trace.

(Also same content as above about default error handling)